# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../python/dont-rebuild-with-equal-time.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We get shared files if the script rebuilds site-package files with
equal time (e.g. often happening for paragui and only depends on the
package's build script or machine speed to happen).

  - Rene Rebe <rene@exactcode.de>

--- Python-2.3.2/Lib/compileall.py.orig	2004-01-10 00:36:26.000000000 +0100
+++ Python-2.3.2/Lib/compileall.py	2004-01-10 00:36:45.000000000 +0100
@@ -58,7 +58,7 @@
                 ftime = os.stat(fullname).st_mtime
                 try: ctime = os.stat(cfile).st_mtime
                 except os.error: ctime = 0
-                if (ctime > ftime) and not force: continue
+                if (ctime >= ftime) and not force: continue
                 if not quiet:
                     print 'Compiling', fullname, '...'
                 try:
